tests/network-tests/src/flows/membership/updatingProfile.ts

```diff
@@ -18,15 +18,31 @@ export default async function updatingProfile({ api, query }: FlowProps): Promis
   const updates: MemberProfileData[] = [
     // Partial updates
     // FIXME: Currently handle always need to be provided, see: https://github.com/Joystream/joystream/issues/2503
-    { handle: 'New handle 1', name: 'New name' },
-    { handle: 'New handle 2' },
+    {
+      handle: 'New handle 1',
+      name: 'New name',
+      isVerifiedValidator: false,
+      validatorAccount: 'validator address',
+    },
+    {
+      handle: 'New handle 2',
+      isVerifiedValidator: false,
+      validatorAccount: 'validator address',
```

Comment:
So it actually test something:
```suggestion
      validatorAccount: 'validator address 2',
```